File: hw/ayuquina/u1/hw11ObjectInFile/ObjectInFilePy/ec_edu_espe_objectinfilepy_model/AutoMobile.py
class AutoMobile():    
    def __init__ (self, id, type_of_automobile, wheels, is_running):

        self.id = id
        self.type_of_automobile = type_of_automobile
        self.wheels = wheels
        self.is_running = is_running

    #toString    
    def __str__(self) -> str:
        return f"AutoMobile (id={self.id}, Type Of Automobile= {self.type_of_automobile}, Wheels={self.wheels}, Is running={self.is_running} )"

    def get_Id(self):
        return self.__id

    def get_type_of_automobile(self):
        return self.__type_of__automobile
    
    def get_is_running(self):
        return self.__is_running

    def get_wheels(self):
        return self.__wheels

File: hw/ayuquina/u1/hw11ObjectInFile/ObjectInFile/src/ec/edu/espe/objectinfile/model/AutoMobile.java
package ec.edu.espe.objectinfile.model;

import java.io.Serializable;

/**
 *
 * @author Danny Ayuquina, LogicLegion, DCCO-ESPE
 */
public class AutoMobile implements Serializable{
    private int id;
    private String typeOfAutoMobile;
    private int numberOfWheels;
    private boolean inMotion;

    public AutoMobile(int id, String typeOfAutoMobile, int numberOfWheels, boolean inMotion) {
        this.id = id;
        this.typeOfAutoMobile = typeOfAutoMobile;
        this.numberOfWheels = numberOfWheels;
        this.inMotion = inMotion;
    }

    @Override
    public String toString() {
        return "AutoMobile{" + "id=" + id + ", typeOfAutoMobile=" + typeOfAutoMobile + ", numberOfWheels=" + numberOfWheels + ", inMotion=" + inMotion + '}';
    }
    

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTypeOfAutoMobile() {
        return typeOfAutoMobile;
    }
    
    public void setTypeOfAutoMobile(String typeOfAutoMobile) {
        this.typeOfAutoMobile = typeOfAutoMobile;
    }

    public int getNumberOfWheels() {
        return numberOfWheels;
    }

    public void setNumberOfWheels(int numberOfWheels) {
        this.numberOfWheels = numberOfWheels;
    }

    public boolean isInMotion() {
        return inMotion;
    }

    public void setInMotion(boolean inMotion) {
        this.inMotion = inMotion;
    }
}
